code/modules/medical/patientprocessor.dm

```diff
@@ -0,0 +1,21 @@
+/obj/machinery/medical/patient_processor
+	name = "patient processor"
+	desc = "A device that uses advanced AutoDoc technology to heal patients."
+	icon = 'icons/obj/machines/mining_machines.dmi'
+	icon_state = "unloader"
+	density = 1
+	anchored = 1
+	var/input_dir = 1
+	var/output_dir = 2
+
+/obj/machinery/medical/patient_processor/process()
```

Comment:
process does not check to see if it is powered [it also doesn't consume power, but that's probably fine as an adminbus toy]

Reply:
Power running out in medbay is a very common problem, caused by another game design flaw of the station requiring power in the first place, which prevents people from getting back into the round as they should. This is completely intentional.